### PR TITLE
Update URL for email events

### PR DESCRIPTION
**Pull request by brainflake**

### CODE REVIEW

**Comment by tejasmanohar** [comment]
Since this is a new endpoint, let's make sure it's pushed out with a breaking change. Preferably a separate function because events by campaign is still a valid use case.

**Comment by brainflake** [comment]
I changed this b/c I thought they changed the endpoint... It should still grab events by campaign. I'll take another look if that's not the case

**Comment by tejasmanohar** [comment]
Hm, don't we already have the get all events endpoint [here](https://github.com/brainflake/node-hubspot/blob/fix-email-events/lib/client.js#L198)? I think I pr'd that earlier haha

**Comment by tejasmanohar** [comment]
if you want to nest that under campaign, fine w/ me. just release as a semver breaking change (major.minor.patch, minor++) & add a changelog.md / history.md file if possible so it's easy for users to migrate

**Comment by brainflake** [comment]
Ah, you are absolutely right @tejasmanohar. I think Hubspot changed up the URLs awhile ago and merged the campaign specific events endpoint into the email events endpoint.

**Comment by brainflake** [comment]
So yeah, I've been putting everything in the Email Events section of the API in the campaigns object here. I guess it's just an organizational decision, but maybe we should consolidate?

I'm fine w/ rearranging things and releasing a breaking change. But this PR (as far as I can tell) is not a breaking change, just a fix since the URL changed for grabbing events by campaign id.

**Comment by tejasmanohar** [comment]
Sounds good, as long as `GET /email/public/v1/events` is only exposed via 1 function (remove the other one) & this is released as a breaking change. Preferably let's add a Changelog/History.md file w/ brief bullets for each release going forward as well :)

**Comment by brainflake** [comment]
Yeah, sounds good to me. I'll send an updated PR when I get a sec

**Comment by brainflake** [comment]
@tejasmanohar just updated to remove the campaigns.tracking object and pull your events method into campaigns. I think this more closely aligns w/ the API (from the docs at least). Any thoughts?

File: lib/client.js [hunk 1]
```diff
@@ -212,11 +198,7 @@ function Client(opts) {
 
       sendRequest(call, cb);
     },
-    tracking: campaignTracking
-  };
-
-  var events = {
-    get: function(options, cb) {
+    events: function(options, cb) {
```
Comment (tejasmanohar) [inline lib/client.js]: why name this function `events` rather than `get`?

**Comment by tejasmanohar** [comment]
lgtm

**Comment by brainflake** [comment]
sweet

**Pull request merged**